<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KitapIslemleri</class>
 <widget class="QDialog" name="KitapIslemleri">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="data_table">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>321</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tüm Kitaplar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sil">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>220</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Sil</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Seçilen Kitabın Ödünç Alınma Durumu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>290</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Seçilen Kitabın Daha Önceden Ödünç Alınma Durumu</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_oad">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>241</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_doad">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>340</y>
     <width>241</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>221</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Kitap No</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Kitap Ad</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Stok</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="input_no">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_ad"/>
      </item>
      <item>
       <widget class="QLineEdit" name="input_stok"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>170</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="btn_kayit">
      <property name="text">
       <string>Yeni Kayıt</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_guncelle">
      <property name="text">
       <string>Güncelle</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>input_no</tabstop>
  <tabstop>input_stok</tabstop>
  <tabstop>btn_guncelle</tabstop>
  <tabstop>data_table</tabstop>
  <tabstop>table_oad</tabstop>
  <tabstop>btn_kayit</tabstop>
  <tabstop>table_doad</tabstop>
  <tabstop>btn_sil</tabstop>
  <tabstop>input_ad</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
